src/lib/stores/workflow-run.ts

```diff
@@ -35,12 +38,26 @@ const updateWorkflowRun: StartStopNotifier<{
   workflow: WorkflowExecution;
   workers: GetPollersResponse;
 }> = (set) => {
-  return parameters.subscribe(({ namespace, workflowId, runId }) => {
+  return parameters.subscribe(({ namespace, workflowId, runId, settings }) => {
     if (namespace && workflowId && runId) {
       withLoading(loading, updating, async () => {
         const workflow = await fetchWorkflow({ namespace, workflowId, runId });
         const { taskQueue } = workflow;
         const workers = await getPollers({ queue: taskQueue, namespace });
+
+        // Decode pending activities
+        const pendingActivities = workflow?.pendingActivities ?? [];
```

Comment:
Given that we do this for events in the model, I wonder if this belongs in a pending activities model as well. It would probably be easier to test.